debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/ReplicationMessage.java

```diff
@@ -186,6 +185,17 @@ default boolean isTransactionalMessage() {
         return getOperation() == Operation.BEGIN || getOperation() == Operation.COMMIT;
     }
 
+    default boolean isFiltered() {
+        return false;
+    }
+
+    /**
+     * Returns true if the message is a skipped message.
+     */
+    default boolean isSkippedMessage() {
```

Comment:
What is the difference between these two messages?

Reply:
My bad. `isFiltered` is a redundant and unused method. will remove it.